Documentation/SystemModules.md

```diff
@@ -17,7 +17,8 @@ Lets put this code in a directory called `example`:
 
     $ mkdir example
     $ cd example
-    example$ touch main.swift Package.swift
+    example$ swift package init --type exec
+    example$ touch main.swift
```

Comment:
Thanks.  Yes, testing out the instructions is part of the pre-merge testing I haven't done yet.